src/components/MyAccount/RequestsTab/RequestsTab.test.tsx

```diff
@@ -45,7 +45,7 @@ describe("RequestsTab", () => {
       />
     )
     const rows = getAllByRole("row")
-    expect(rows.length).toBe(3)
+    expect(rows.length).toBe(7)
```

Comment:
The data, as far as I can tell, has 6 holds. Is the heading counted as a row?

Reply:
Right– too unclear? I could switch to "tbody > tr" selector

Reply:
Maybe `within(..."tbody"...).getAllByRole("row")`?

Reply:
`const bodyRows = component.getAllByRole("rowgroup")[1]
    expect(within(bodyRows).getAllByRole("row").length).toBe(4)`
:( but idk what else bc we have no test ids or ids on these